src/main/java/uk/co/ramp/covid/simulation/population/Infant.java

```diff
@@ -7,7 +7,7 @@
 
 public class Infant extends Person {
 
-    private boolean goesToNursery;
+    boolean goesToNursery;
```

Comment:
I think it's best to leave this private and add a getter (to make it clear this value can be queried, but not updated)

Reply:
Agreed